### ISSUE TITLE
[BUG] [Linter] Rule T028 should only catch if/for statements

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#139]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Issue

```bash
echo '<a href="{% url 'home' %}">Home</a>' | djlint -
```
should not be triggering rule T028. Only if/else/for should trigger it.

**Comment by christopherpickering** [comment]
@yaegassy I think this commit will fix your problem. Let me know if something is not right.

**Comment by yaegassy** [comment]
Thanks for the fix!
